src/Symfony/Component/Messenger/Stamp/HandledStamp.php

```diff
@@ -23,8 +23,19 @@
  */
 final class HandledStamp implements StampInterface
 {
+    /**
+     * @var mixed
+     */
     private $result;
+
+    /**
+     * @var string
```

Comment:
We don't add `@var` phpdoc on properties in Symfony, especially when the constructor allows IDEs to infer it already.